include/constants/action.h

```diff
@@ -0,0 +1,44 @@
+#ifndef GUARD_CONSTANTS_ACTION_H
+#define GUARD_CONSTANTS_ACTION_H
+
+enum Action
```

Comment:
```suggestion
enum DungeonAction
```

Since this is specific to Dungeon, makes more sense to prefix it

Reply:
Renamed the enum selector, file, and all action enums.